shared/util/quit-helper.desktop.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import electron from 'electron'
+
+const app = electron.app || electron.remote.app
+const BrowserWindow = electron.BrowserWindow || electron.remote.BrowserWindow
+
+type WindowContext = {
```

Comment:
:sheep: is there a less-vertically-wasteful formatting that works okay with eslint here? :)